schema.sql

```diff
@@ -71,4 +71,14 @@ CREATE TABLE visits (
 	date_of_visit DATE,
 	FOREIGN KEY (animal_id) REFERENCES animals(id),
 	FOREIGN KEY (vet_id) REFERENCES vets(id)
-);
\ No newline at end of file
+);
+
+-- Perfomance audit exercise
+-- Create a non clustered index for animal id on  visits table
+CREATE INDEX animal_id_asc ON visits(animal_id ASC);
+
+-- Create a non clustered index for vet id on visits table
+CREATE INDEX vet_id_asc ON visits(vet_id ASC);
+
```

Comment:
**Optional**
- Even if the execution times difference is very small in your case, I can clearly see that you had some fun with the `explain analyze` and I am sure that in the future you will know how to start the investigation for some slow DB queries :)